ci/scripts/check-bazel-repos.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+GIT_REPOS=$(./bazelisk.sh query "kind(new_git_repository, //external:*) union kind(git_repository, //external:*)")
```

Comment:
I feel like this obscures the fact that `kind`'s first arg is a regex. I'd embrace it and just say `"(new_)?git_repository"` :)

Reply:
+1